javascript/src/test/components/Statistics/AggregateStatisticsForm.test.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { render, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import AggregateStatisticsForm from "main/components/Statistics/AggregateStatisticsForm";
+jest.mock("isomorphic-unfetch"); 
+
+describe("AggregateStatisticsForm tests", () => {
+
+    test("renders without crashing", () => {
+        render(<AggregateStatisticsForm />);
+    });
+
+    test("when I select a start quarter, the state for start quarter changes", () => {
+        const { getByTestId } = render(<AggregateStatisticsForm />);
+        const selectQuarter = getByTestId("select-start-quarter")
+        userEvent.selectOptions(selectQuarter, "3");
+        expect(selectQuarter.value).toBe("3");
+    });
+
+    test("when I select a end quarter, the state for end quarter changes", () => {
+        const { getByTestId } = render(<AggregateStatisticsForm />);
+        const selectQuarter = getByTestId("select-end-quarter")
+        userEvent.selectOptions(selectQuarter, "3");
+        expect(selectQuarter.value).toBe("3");
+    });
+
+    test("when I select a start quarter year, the state for start quarter year changes", () => {
+        const { getByTestId } = render(<AggregateStatisticsForm />);
+        const selectQuarter = getByTestId("select-start-year")
+        userEvent.selectOptions(selectQuarter, "2019");
+        expect(selectQuarter.value).toBe("2019");
+    });
+
+    test("when I select a end quarter year, the state for end quarter year changes", () => {
+        const { getByTestId } = render(<AggregateStatisticsForm />);
+        const selectQuarter = getByTestId("select-end-year")
+        userEvent.selectOptions(selectQuarter, "2019");
+        expect(selectQuarter.value).toBe("2019");
+    });
+
+    test("when I click submit with a good date range, the right stuff happens", async () => {
```

Comment:
It'd be good to have more informative names for this and the following test- what is the right stuff that we should expect to happen?